src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2030,19 +2030,19 @@ internal static string ConcatenateStringPathArguments(CommandElementAst stringAs
             {
```

Comment:
@powercode, your last commit had 7 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - XmlFileContentResource - Good case
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1: line 348
348:             $resource | Should -Not -BeNullOrEmpty
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - XmlFileContentResource - Module Name case mismatch
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1: line 348
348:             $resource | Should -Not -BeNullOrEmpty
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - xmlfilecontentresource - Resource name case mismatch
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1: line 348
348:             $resource | Should -Not -BeNullOrEmpty
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - XmlFileContentResource from XmlContentDsc - Good case
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1: line 323
323:             $resource | Should -Not -BeNullOrEmpty
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - XmlFileContentResource from xmlcontentdsc - Module Name case mismatch
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1: line 323
323:             $resource | Should -Not -BeNullOrEmpty
```